engine/table/src/main/java/io/deephaven/engine/table/impl/SourcePartitionedTable.java

```diff
@@ -186,20 +183,43 @@ private RowSet sortAndAddLocations(@NotNull final Stream<TableLocation> location
                 result.manage(constituentTable);
             });
             return initialLastRowKey == lastInsertedRowKey.longValue()
-                    ? RowSetFactory.empty()
-                    : RowSetFactory.fromRange(initialLastRowKey, lastInsertedRowKey.longValue());
+                   ? RowSetFactory.empty()
+                   : RowSetFactory.fromRange(initialLastRowKey + 1, lastInsertedRowKey.longValue());
         }
 
         private Table makeConstituentTable(@NotNull final TableLocation tableLocation) {
-            return applyTablePermissions.apply(new PartitionAwareSourceTable(
+            final PartitionAwareSourceTable constituent = new PartitionAwareSourceTable(
                     constituentDefinition,
                     "SingleLocationSourceTable-" + tableLocation,
                     RegionedTableComponentFactoryImpl.INSTANCE,
                     new SingleTableLocationProvider(tableLocation),
-                    refreshSizes ? refreshCombiner : null));
+                    refreshSizes ? refreshCombiner : null);
+
+            // These can't be systemic or when they get notified on error,  they will crash the worker.
+            constituent.setAttribute(Table.SYSTEMIC_TABLE_ATTRIBUTE, false);
```

Comment:
We need to be able to determine this based on the systematic attribute of the whole table, or our context not just force it to false.  A systemic SourcePartitionedTable should propagate that attribute to the constituents; one that is not systemic should not.